mailutils.php

```diff
@@ -82,3 +83,74 @@ function mailutils_civicrm_navigationMenu(&$menu) {
   ));
   _mailutils_civix_navigationMenu($menu);
 }
+
+/**
+ * Implements hook_coreResourceList
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_coreResourceList
+ */
+function mailutils_civicrm_coreResourceList(&$list, $region) {
+  CRM_Core_Resources::singleton()->addStyleFile('mailutils', 'css/mailutils.css', 0, 'page-header');
+}
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_buildForm
+ */
+function mailutils_civicrm_buildForm($form_name, &$form) {
+  switch ($form_name) {
+    case 'CRM_Activity_Form_ActivityView': {
```

Comment:
I found one more activity form in core 🤦 : `CRM_Activity_Form_Activity` (via `/civicrm/activity?reset=1&action=view&id=...`, which of course is different from `/civicrm/activity/view?action=view&reset=1&id=708&cid=307](/civicrm/activity/view?action=view&reset=1&id=...`)

Reply:
Sorry, one more - there's also `CRM_Case_Form_Activity` via `/civicrm/case/activity?reset=1&cid=...&caseid=...&id=...&action=view`

😭